src/Factories/ReaderFactory.php

```diff
@@ -31,12 +31,12 @@ public static function make($type, $import): ReaderInterface
     protected static function makeUnconfiguredReader($type)
     {
         if (Simplesheet::TSV === $type) {
-            return SpoutReaderFactory::createFromType(Type::CSV)
+            return SpoutReaderFactory::createCSVReader(Type::CSV)
                 ->setFieldDelimiter("\t")
                 ->setShouldPreserveEmptyRows(true);
         }
 
-        return SpoutReaderFactory::createFromType($type)->setShouldPreserveEmptyRows(true);
+        return SpoutReaderFactory::createCSVReader()->setShouldPreserveEmptyRows(true);
```

Comment:
This will prevent creating readers for types other then CSV and TSV. So ODS and XLSX would get wrong reader